template_credentials.go

```diff
@@ -0,0 +1,78 @@
+package transloadit
+
+import (
+	"context"
+)
+
+// Template contains details about a single template.
```

Comment:
Comment should read template credential not template.

Reply:
done